examples/lammps/isof-water/input.xml

```diff
@@ -8,7 +8,7 @@
   <prng>
     <seed>32345</seed>
   </prng>
-  <ffsocket name='lmpserial' mode='unix'>
+  <ffsocket name='qtip4pf' mode='unix' pbc='false'>
```

Comment:
is pbc=false needed here?

Reply:
This tag controls if ipi applies pbc before passing the data to the client.
Since qtip4pf built-on driver is not pbc proof, it is safer to set it to false.

Reply:
Yes, I am fully aware about what the flag does. But now I see that again here we have this naming problem (you are calling qtip4p from the driver, but the example is in lammps folder). Let's try to find a better solution for this, this will become very confusing to future users.